### PR TITLE
Add String.Reverse/reverse.

### PR DESCRIPTION
**Pull request by mjuksel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -531,6 +531,18 @@ const result = upperCase(str)
 //    ^ 'HELLO WORLD'
 ```
 
+### reverse
+
+This function truncates string if it's longer than the given maximum string length. The last characters of the truncated string are replaced with the omission string which defaults to "...".
```
Comment (gustavoguichard) [inline README.md:536]: This is from the `truncate` method 😬

**Comment by gustavoguichard** [comment]
Thank you for your contribution @mjuksel !!
Your PR is very welcome!
I left a comment to adjust the README ;)

File: README.md (hunk 1 above)
Comment (mjuksel) [reply README.md]: Wups, fixed 🤣

**Comment by mjuksel** [comment]
> I left a comment to adjust the README ;)

Fixed 👍

**Review by gustavoguichard — APPROVED** [review]
LGTM

**Comment by gustavoguichard** [comment]
@all-contributors please add @mjuksel for code, ideas.

**Pull request closed without merging**